javascript/callbacks.js

```diff
@@ -58,7 +61,7 @@ export const afterDOMUpdate = event => {
         element: reflexElement,
         event,
         data: promise.data,
-        payload: event.detail.payload
+        payload

```

Comment:
Would it make sense to use a named argument for payload similar to what we do with data?

```js
data: promise.data,
payload: payload
```